practice-app/backend/api/migrations/0001_initial.py

```diff
@@ -12,13 +12,14 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.CreateModel(
-            name='Country',
+            name='Poll',
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('name', models.CharField(max_length=50)),
-                ('capital', models.CharField(max_length=50)),
-                ('population', models.IntegerField(default=0)),
-                ('gdp', models.IntegerField(default=0)),
+                ('question', models.CharField(max_length=1000)),
+                ('firstOption', models.CharField(max_length=300)),
+                ('secondOption', models.CharField(max_length=300)),
+                ('thirdOption', models.CharField(max_length=300, null=True)),
+                ('fourthOption', models.CharField(max_length=300, null=True)),
             ],
```

Comment:
Aga bu dosyaya niye dokundun ki onu anlayamadım. Direk 2. migrationa koysaydın ya bu fieldları

Reply:
Migrationlar aşırı kritik değil onlar database i inşa etmek için şu an database e hep ekleme yapılacak şimdilik çalışıyorsa migrations a bir şey demeyelim

Reply:
Migrationın ne olduğunu biliyorum ama migration sıralamaları önemlidir ki migrationı boşver yapılan şey mantıksız gereksiz kod değişikliği olmuş silinen bütün satırlar bir daha eklenmiş. 100 eklenmesi gerekirken 200 eklenip 100 silinmiş tarzı

Reply:
Onları bütün objectler yazılınca tek çatı altında toplayalım şu anda herkes o kısmı baştan yazacak zaten